themes/default/document.css

```diff
@@ -5,14 +5,25 @@
   padding: 10px 30px;
 }
 
-.ct-document-header {
+.ct-document-header, .ct-document-footer {
   align-items: center;
-  border-bottom: 1px solid #eee;
   display: flex;
   margin-bottom: 30px;
   padding: 10px 0 20px;
 }
 
+.ct-document-header {
+  border-bottom: 1px solid #eee;
+  margin-bottom: 30px;
+  padding: 10px 0 20px;
```

Comment:
These two rows is duplicated in both rules. They only need to be in one of them, right?